test/plugin/test_in_tail.rb

```diff
@@ -2648,6 +2648,21 @@ def test_no_warn_with_directory_permission
     end
   end
 
+  def test_other_user_owned_files
```

Comment:
Could you add a comment to refer the issue?

Reply:
Thanks.
I added at https://github.com/fluent/fluentd/pull/5161/commits/1d7abe5b44770b9720066dc7f7383414c0ecbfb5